rubin_scheduler/scheduler/utils/footprints.py

```diff
@@ -71,17 +70,9 @@ def make_rolling_footprints(
         nslice=6.
     n_constant_start : `int`
         The number of constant non-rolling seasons to start with. Anything
-        less than 2 will start rolling too early near Y1. If uniform rolling
-        is turned off, the lowest value should be 3. Defaults to None which
-        selects the correct lowest value based on the value of uniform.
+        less than 2 will start rolling too early near Y1. Defaults to 2.
```

Comment:
We have a partial season over the whole sky (season -1) and then a full season over most of the sky (season 0) but then rolling starts within the last 3 months of this -- within season 0 in "visibility/proper seasons" but at the start of season 2 if you start counting at 0 and offset by 90 degrees, as the footprint season does.  So I guess this is accurate in counting in terms of "footprint seasons", but it would be great to align these at some point (instead of going ramp then constant in the season footprint, doing constant - ramp - constant).